pyconcz/programme/models.py

```diff
@@ -167,3 +167,15 @@ def __init__(self):
 
     def get_group_by_cols(self):
         return []
+
+class Utility(models.Model):
+    title = models.CharField(max_length=200, verbose_name='Title')
+    description = models.TextField(blank=True, null=True)
+    url = models.CharField(max_length=255, blank=True, null=True)
```

Comment:
URL field neumožňuje relativní url, jestli se nepletu.
https://code.djangoproject.com/ticket/10896
vadí to protože pokud tam budou absolutní URL, tak při vývoji/na betě ty odkazy povedou na produkci…

Reply:
@benabraham tak co keby sa ten url field zostal ako je ale pred zobrazenim uzivatelovi by sa zistilo ci je to relative, a ked ano pridala by sa tam dynamicky spravna domena, co na to hovoris?

Reply:
Nápad @mcurlej mi zní dobře. Budeme tak mít možnosti odkazovat se sami na sebe funkční i při vývoji a také možnost odkázat se na externí adresu.

Reply:
Mě to zní zbytečně komplexně ve srovnání se stringem, ale když to tak uděláš nebudu ti bránit.